vmdb/app/models/ems_refresh/parsers/kubernetes.rb

```diff
@@ -98,6 +98,9 @@ def parse_node(node)
 
     def parse_service(service)
       new_result = parse_base_item(service)
+      if new_result[:ems_ref].nil? # Typically this happens for kubernetes services
+        new_result[:ems_ref] = "#{new_result[:namespace]}_#{new_result[:name]}"
```

Comment:
Doesn't have to be here but does it make sense for `parse_base_item(service)` to have this logic?  In other words, let parse_base_item populate the `:ems_ref` key's value with the namesspace + name value if it's nil.